aspnetcore/blazor/security/blazor-web-app-with-oidc.md

```diff
@@ -53,7 +53,7 @@ The `BlazorWebAppOidc.http` file can be used for testing the weather data reques
 This section explains how to configure the sample app.
 
 > [!NOTE]
-> For Microsoft Entra ID and Azure AD B2C, you can use <xref:Microsoft.Identity.Web.AppBuilderExtension.AddMicrosoftIdentityWebApp%2A> from [Microsoft Identity Web](/entra/msal/dotnet/microsoft-identity-web/) ([`Microsoft.Identity.Web` NuGet package](https://www.nuget.org/packages/Microsoft.Identity.Web), [API documentation](<xref:Microsoft.Identity.Web?displayProperty=fullName>)), which adds both the OIDC and Cookie authentication handlers with the appropriate defaults. The sample app and the guidance in this section doesn't use Microsoft Identity Web. The guidance demonstrates how to configure the OIDC handler *manually* for any OIDC provider. For more information on implementing Microsoft Identity Web, see the linked resources.
+> For Microsoft Entra ID, Entra ID for customers and Azure AD B2C, you can use <xref:Microsoft.Identity.Web.AppBuilderExtension.AddMicrosoftIdentityWebApp%2A> from [Microsoft Identity Web](/entra/msal/dotnet/microsoft-identity-web/) ([`Microsoft.Identity.Web` NuGet package](https://www.nuget.org/packages/Microsoft.Identity.Web), [API documentation](<xref:Microsoft.Identity.Web?displayProperty=fullName>)), which adds both the OIDC and Cookie authentication handlers with the appropriate defaults. The sample app and the guidance in this section doesn't use Microsoft Identity Web. The guidance demonstrates how to configure the OIDC handler *manually* for any OIDC provider. For more information on implementing Microsoft Identity Web, see the linked resources.
```

Comment:
I don't understand this change, and it feels like it breaks our internal guidelines for referring to these products. Can you clarify this change?

Reply:
Yeah ... so this one ... I don't think we'll be able to take this suggestion. However, let me hear your reason for the change, and we can see what's what with it.

Reply:
Hi @guardrex Microsoft Entra ID for customers is the name of the new B2C product. This is a different MS identity provider and has a different setup. It will replace Azure AD B2C.

https://learn.microsoft.com/en-us/entra/external-id/customers/

Formally known as CIAM.

Reply:
Let me know if I should remove this.

We have 3 different products for IAM now:

- Microsoft Entra ID (B2B)
- Microsoft Entra ID for customers (B2C, CIAM to replace Azure AD B2C)
- Azure AD B2C

Reply:
I see ... I haven't heard about this yet, and it's a bit confusing to me at the moment.

@tdykstra @wadepickett @Rick-Anderson ... Are we advised to update the naming of Azure AD B2C? If so, I hope to see their official (perhaps internal) guidance on how the product name for B2C is shifting.

Reply:
@guardrex  how did I miss some. See #30185  and https://learn.microsoft.com/entra/fundamentals/new-name

Reply:
Thx @Rick-Anderson ... That was the latest info that I had.

@damienbod ... Thanks for all of the updates. I'll merge without the first suggestion.